.devcontainer/Dockerfile

```diff
@@ -1,9 +1,15 @@
 FROM mcr.microsoft.com/vscode/devcontainers/python:0-3.11-bullseye
 
 RUN apt-get update && apt-get install -y apt-transport-https gnupg2 procps less curl
+
 RUN wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | apt-key add -
 RUN echo "deb https://artifacts.elastic.co/packages/7.x/apt stable main" | tee /etc/apt/sources.list.d/elastic-7.x.list
-RUN apt-get update && apt-get install elasticsearch
```

Comment:
We were previously installing the minimum supported ES version into the container. I'm not opposed to changing the default, but perhaps we should leave the command commented out if others want to more easily swap versions?
Disregard if we plan to move forward with https://github.com/legrego/homeassistant-elasticsearch/issues/230 sooner rather than later.

Reply:
I'll undo this change as I'm already pretty far in having a pr for running ES in docker